sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/RedistributeTest.java

```diff
@@ -289,13 +284,12 @@ public void testRedistributeAfterFixedWindowsAndGroupByKey() {
   @Test
   @Category(ValidatesRunner.class)
   public void testRedistributeAfterSlidingWindowsAndGroupByKey() {
-
     PCollection<KV<String, Iterable<Integer>>> input =
         pipeline
             .apply(
                 Create.of(GBK_TESTABLE_KVS)
                     .withCoder(KvCoder.of(StringUtf8Coder.of(), VarIntCoder.of())))
-            .apply(Window.into(FixedWindows.of(Duration.standardMinutes(10L))))
+            .apply(Window.into(SlidingWindows.of(Duration.standardMinutes(10L))))
```

Comment:
the original tests for slidingWindow was incorrect and likely never worked. After this change, test fails on portable-flink-runner validation test suite: https://github.com/apache/beam/actions/runs/10025426477/job/27708578006#step:6:202

we can leave it as is and figure it out later

Reply:
@Abacn  Thank you for your response! I notice there are few failing checks relevant to style checks, I will fix them soon